app/routes/goal_routes.py

```diff
@@ -1 +1,113 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, abort, make_response
+from app.models.goal import Goal
+from app.routes.route_utilities import validate_model
+from ..db import db
+from app.models.task import Task
+goals_bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
+
+@goals_bp.post("")
+def create_goal():
+
+    request_body = request.get_json()
+
+    if "title" not in request_body:
+        return {"details": "Invalid data"}, 400
+
+    new_goal = Goal(title=request_body["title"])
+    db.session.add(new_goal)
+    db.session.commit()
+
+    response = {
+        "goal": {
+        "id": new_goal.id,
+        "title": new_goal.title
+    }
+    }
```

Comment:
I noticed you don't have a to_dict method for your Goal model. This code seems like it will probably be repeated, so a `to_dict` could make it a bit more concise!